src/d3a/models/market/market_structures.py

```diff
@@ -25,8 +26,14 @@
 Clearing = namedtuple('Clearing', ('rate', 'energy'))
 
 
+def my_converter(o):
+    if isinstance(o, datetime):
+        return o.__str__()
```

Comment:
It would be much better if you did not handle it this way. What you can alternatively do is to use `isostring` in order to change only the fields you want to, and not use a custom formatter that converts to string. In general, isostrings are easily converted back to DateTimes, strings of DateTimes not so easily.